src/components/App/App.js

```diff
@@ -32,9 +40,31 @@ class App extends React.Component {
 		}
 	};
 
+	showNotification = message => {
+		this.setState(
+			{notificationMessage: message, showNotification: true},
+			() => {
+				setTimeout(() => {
+					this.setState({showNotification: false});
+				}, 5000);
+			}
+		);
+	};
+
 	render() {
+		if (this.props.slackUserFetchError)
+			setTimeout(() => {
+				this.props.setSlackUserFetchError(false);
```

Comment:
Render is supposed to be pure and not contain side effects (besides some new and upcoming APIs), could we move this check to `componentDidUpdate`?

Reply:
Actually, would it make more sense to have state manage this rather than wiping props (there's still a slack error)

E.g. if there wasn't an error, but now there is, then show the error and hide it after 5 seconds